Lombiq.Tests.UI/Services/HtmlValidationConfiguration.cs

```diff
@@ -88,7 +92,18 @@ public HtmlValidationConfiguration WithRelativeConfigPath(params string[] pathSe
     public static readonly Func<HtmlValidationResult, Task> AssertHtmlValidationOutputIsEmptyAsync =
         validationResult =>
         {
-            validationResult.Output.ShouldBeEmpty();
+            // Keep supporting cases where output format is not set to JSON.
+            if (validationResult.Output.Trim().StartsWith('[') ||
+                validationResult.Output.Trim().StartsWith('{'))
```

Comment:
I guess there isn't a fancier way to check this, I looked into it but I haven't came up with anything either. What do you think?

Reply:
Unfortunately we don't have access to the options here so we can't directly check, I haven't been able to think of a nicer way of doing it.